spyder/plugins/pythonpath/widgets/tests/test_pathmanager.py

```diff
@@ -277,6 +298,14 @@ def test_buttons_state(qtbot, pathmanager, tmpdir):
     pathmanager.remove_path(True)
     assert not pathmanager.button_ok.isEnabled()
 
+    # Check prioritize button
+    assert pathmanager.prioritize_button.isEnabled()
+    assert not pathmanager.prioritize_button.isChecked()
+    pathmanager.prioritize_button.animateClick()
+    qtbot.waitUntil(pathmanager.prioritize_button.isChecked)
+    assert pathmanager.prioritize_button.isChecked()
+    assert pathmanager.button_ok.isEnabled()
+
```

Comment:
I think a test for the new prioritize button is missing. Could you add one?

Reply:
So the state of the OK button is tested with respect to changes to the prioritize button in `test_buttons_state`. The Pythonpath Manager doesn't actually do anything with the prioritize state except to pass its value via signal, i.e. it doesn't do any path sorting, so I'm not sure what should be tested here. `spyder-kernels` will actually do the sorting, and this is tested in `test_console_kernel.py`, added in the companion PR. Is there something in particular you had in mind to test?

Reply:
Ok, but we need to check that the Spyder UI actually changes the priority of paths in the console at least (it's harder to check in the LSP).

For that then I suggest to extend `test_PYTHONPATH_in_consoles`, by clicking the prioritize button in the PPM dialog and then check in the console that the paths handled by it are at the front of `sys.path`.